src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppAppRunner/AspNetAppAppRunner.csproj

```diff
@@ -26,7 +26,7 @@
     <!-- CDK Construct Library dependencies -->
     <PackageReference Include="Amazon.CDK.Lib" Version="2.43.1" />
 
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.7" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
```

Comment:
Throughout, for any `PackageReference` changes please update both the internal license tracking tool and then regenerate THIRD_PARTY_LICENSES.

Reply:
Done